conf/Hsapiens/genomes.config

```diff
@@ -18,6 +18,8 @@ params {
             fai                 = "${params.genomes_base}/Hsapiens/GRCh38.p14/seq/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set.fna.fai"
             fasta               = "${params.genomes_base}/Hsapiens/GRCh38.p14/seq/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set.fna"
             dict                = "${params.genomes_base}/Hsapiens/GRCh38.p14/seq/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set.dict"
+            sdf                 = "${params.genomes_base}/Hsapiens/GRCh38.p14/seq/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set_sdf.tar.gz"
+            strtablefile        = "${params.genomes_base}/Hsapiens/GRCh38.p14/seq/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set_strtable.zip"
```

Comment:
What about https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-configs/blob/c4b7cf09becaa0a532e73beda17067db9cf43ba7/conf/Hsapiens/genomes.config#L56 ?

Reply:
Oh I didn't know it was already in there, thanks!